src/amo/components/Header/styles.scss

```diff
@@ -0,0 +1,60 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+.Header {
+  background: $header-base-color;
+  min-height: 100px;
+  position: relative;
+}
+
+.Header-content {
+  padding: 10px 0;
+  text-align: center;
+
+  @include respond-to(mobile) {
```

Comment:
As mentioned above I'm not sure how I feel about that: this is something that only happens on the mobile breakpoint–were we to add new breakpoints we would need to go back and change everything that uses `respond-to(desktop)` even though this is something we only want to happen on very small screens.